cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/query/QueryResult.java

```diff
@@ -195,7 +194,8 @@ private DocumentRevision projectFields(List<String> fields,
         revBuilder.setBody(DocumentBodyFactory.create(body));
         revBuilder.setDeleted(rev.isDeleted());
         revBuilder.setAttachments(new ArrayList<Attachment>(rev.getAttachments().values()));
-        revBuilder.setSequence(rev.getSequence());
+        // TODO sequence is internal anyway, why should we expose it out?
+        //revBuilder.setSequence(rev.getSequence());
```

Comment:
We shouldn't expose it outside the datastore package, I think we should be safe to remove these two lines.